volontulo/templates/common/header.html

```diff
@@ -14,7 +14,7 @@
         <div class="col-xs-12">
             <h1 class="navbar-header">
                 <a href="{% url 'index' %}">
-                    <img src="{% static "volontulo/img/volo_logo.svg" %}" alt="Volontulo">
+                    <img src="{% static "volontulo/img/volo_logo.svg" %}" alt="Volontulo" />
                     Volontulo
```

Comment:
Thus I removed lonely string Volontulo and there is still alt attribute of image.